src/rpcmining.cpp

```diff
@@ -61,6 +66,40 @@ UniValue getmininginfo(const UniValue& params, bool fHelp)
         obj.pushKV("kernel-diff-sum",MinerStatus.KernelDiffSum);
     }
 
+    int64_t nMinStakeSplitValue = 0;
+    double dEfficiency = 0;
+    SideStakeAlloc vSideStakeAlloc = {};
+
```

Comment:
Speculation on my part: instead of calling the parsing function here, read the global variables from miner.

Reply:
These are not global. I purposely did not make them global. I would rather not create more globals for this unless we really need to. More globals = more locks and more headaches. The

Reply:
Good point. I thought the values were stored in globals.

Reply:
But now we would have rpc that just returns info from config file. This is quite unusual, because RPCs usually return some runtime or chain data, not stuff that could be read from config. Aha I see that the values are altered on load, so having the output in rpc to see what is actually going on is useful.

Reply:
Yes, because there are clamps, etc... We may have to go to globals when we implement the UI table to allow changing on the fly... but that is a larger discussion that belongs in the new UI work I think...

Reply:
Actually, something easy to add to the rpc would be the UTXO size calculated for the current specified efficiency and diff average. Basically line 830 and 831 of miner.cpp...

        nDesiredStakeOutputValue = G * GetAverageDifficulty(160) * (3.0 / 2.0) * (1 / dEfficiency  - 1) * COIN;
        nDesiredStakeOutputValue = max(nMinStakeSplitValue, nDesiredStakeOutputValue);

Reply:
I could just add that in to GetStakeSplitStatusAndParams and put nDesiredStakeOutputValue as an out parameter. The biggest overhead is the diff calculation over 160 blocks.

Reply:
Did it. Let me know if you have any suggestion on what I did. I think it is ok... and didn't have to take any additional locks for GetDifficulty call, since cs_main was already taken at the beginning of the rpc call.